### PR TITLE
better CIM compliance and proper formatted config

### PR DESCRIPTION
**Pull request by micbi-dt**

### CODE REVIEW

File: src/Egnyte_Protect/default/app.conf [hunk 1]
```diff
@@ -3,8 +3,14 @@
 #
 [ui]
 is_visible = 1
+show_in_nav = true
 label = Egnyte Secure & Govern App for Splunk
 
+[install]
+state_change_requires_restart = true
+is_configured = false
+state = enabled
```
Comment (shethankit) [inline src/Egnyte_Protect/default/app.conf:12]: By Default State is always enabled and is_configured is always set to false.

File: src/TA-egnyte-protect/default/props.conf [hunk 2]
```diff
@@ -1,20 +1,200 @@
...
+# [X] DLP       severity - The severity of the DLP event.
+# [X] ALERT     severity - The severity of the network protection event. (critical, high, medium, low, informational)
+# [ ] DLP       severity - The severity of the DLP event.
+EVAL-severity = case(\
+    severity<=2, "informational", \
+    severity>=3 AND severity<=4, "low", \
+    severity>=5 AND severity<=6, "medium", \
+    severity>=7 AND severity<=8, "high", \
+    severity>=9, "critical" \
+)
```
Comment (shethankit) [inline src/TA-egnyte-protect/default/props.conf:140]: We can not update the severity flags according to this as the severity flags needs to be aligned with Egnyte S&G's UI. And the current values are aligned according to the severity is been categorised in the Egnyte UI.

File: src/TA-egnyte-protect/default/app.conf [hunk 3]
```diff
@@ -11,7 +12,8 @@ author = Egnyte Inc
 description = This TA provides interface to ingest incidents from Egnyte Secure & Govern into Splunk.
 
 [ui]
-is_visible = 1
+is_visible = true
+show_in_nav = false
```
Comment (shethankit) [inline src/TA-egnyte-protect/default/app.conf:16]: Egnyte S&G needs to be visible in the Nav so, `show_in_nav` needs to be set to true.

**Review by shethankit — CHANGES_REQUESTED** [review]

File: src/TA-egnyte-protect/default/props.conf (hunk 2 above)
Comment (micbi-dt) [reply src/TA-egnyte-protect/default/props.conf:140]: Splunk CIM has 5 severity levels which should be reflected here... no need to correlate with the source system severity system as this should fit into the overall severity scoring of the SIEM.

File: src/TA-egnyte-protect/default/app.conf (hunk 3 above)
Comment (micbi-dt) [reply src/TA-egnyte-protect/default/app.conf:16]: visible yes but best practice is to make Technical add-ons just accessible via the application management and not take space in the launcher side bar.

**Pull request merged**